src/lib/inputs/InputRadio/input-radio-documentation/input-radio.examples.tsx

```diff
@@ -12,8 +13,42 @@ export const inputRadioExample1: React.FC<any> = () => (
     {formikProps => (
       <>
         Value: {formikProps.values.sampleRadio}
-        <InputRadio id="sampleRadio.yes" label="Yes" name="sampleRadio" value="Yes" formikProps={formikProps} />
-        <InputRadio id="sampleRadio.no" label="No" name="sampleRadio" value="No" formikProps={formikProps} />
+        <InputRadioGroup
+          formikProps={formikProps}
+          legend="Sample Radio"
+          name="sampleRadio"
+          value={formikProps.values.sampleRadio}
```

Comment:
@Carlo-Piantini this value should be assumed if formikProps is passed in. (see other comments about this)

Reply:
Yup, this works! This is resolved.